OpenSim/Simulation/Model/PathPoint.h

```diff
@@ -35,88 +35,76 @@ class PhysicalFrame;
 //=============================================================================
 //=============================================================================
 /**
- * A class implementing a path point.
- *
- * @author Peter Loan
- * @version 1.0
+ * A PathPoint that is stationary with respect to parent's PhysicalFrame
```

Comment:
I will revise the property description. It should not be necessary to know the internal implementation of the `PathPoint` that now uses a `Station`.